docs/UserGuide.md

```diff
@@ -503,21 +509,22 @@ use the `edit` command to input the accurate details._
 >
 > At least one of the optional fields must be provided.
 
-**Examples:** 
+**Examples:**
 
-1. You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
+**Step 1.** You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
```

Comment:
You received an email from Microsoft inviting you to interview for a Frontend Developer role, which differs from the Software Engineer role you initially applied for.


will this phrasing be better